kong/dao/db/cassandra.lua

```diff
@@ -24,8 +24,8 @@ _M.dao_insert_values = {
     return uuid()
   end,
   timestamp = function()
-    -- return time in UNIT millisecond, and PRECISION millisecond 
-    return math.floor(timestamp.get_utc_ms()) 
+    -- return time in UNIT millisecond, and PRECISION millisecond
+    return math.floor(timestamp.get_utc_ms())
```

Comment:
can we avoid these whitespace cleanups. glad we're cleaning up, but it dirties the commit history. we can have a separate style clean commit later.